### ISSUE TITLE
Windows 11 issues (opacity + icons)

### ISSUE THREAD

**Issue by ns8wastaken** [tjackenpacken/taskbar-groups#235]
When I create a folder and add an icon it is simply not displayed in the taskbar and when I change the opacity it doesn't change the opacity in the taskbar either. For the icons I've tried png, jpg, ico...

**Comment by hansnijssen** [comment]
Correct, I have the exact same problem here, icon is simply not showing in taskbar, but it's there all right...(blank square)

**Comment by trxsudo** [comment]
same its broken
